### ISSUE TITLE
Origins won't work for my friend?

### ISSUE THREAD

**Issue by ThePuppetMaster0101** [apace100/origins-fabric#114]
My friend downloaded Origins on what i believe is an air mac, However no matter what version they use it doesn't work on minecraft, we thought it just wouldnt let her pick an origin but after looking in the controls it doesn't appear. It pretty much acts as if its not installed, All her other mods work just fine. We even tried installing extra origins to see if it would let her pick any of thoose but it also didn't work. Is origins specifically made for windows or something? And if so could you please make a version for Mac? If anybody has a solution i would appreciate the help :)

In short:
Origins wont show up on my friends mac even though its the right version.

**Comment by ThePuppetMaster0101** [comment]
Just looked in closed issues and found out its because origins isnt working. I forgot to mention she is invisible too, I dunno how to solve this but i suppose we will just try downloading different versions of origins, if there is something to do with it not working on mac tell me ig

**Comment by apace100** [comment]
The mod should work fine on Mac. I suggest googling for tutorials on how to install Fabric mods on Mac (any generic one will work, and then Origins can be installed like any other mod, which the tutorial will probably tell you how to do).

Regarding the invisibility issue: Please make sure everything is installed correctly and works for everyone in singleplayer. Also make sure that all players and the server have the same version of Origins installed.

If the problem still persists, feel free to reply here and I'll reopen the issue.
